Convert byte array to string (#266)
rogchap/v8go: CHANGELOG.md

## [Unreleased]

### Added

> - Support for [binary strings](https://developer.mozilla.org/en-US/docs/Web/API/DOMString/Binary). `NewStringFromByteArray` takes bytes array as input and uses `String::NewFromOneByte` to convert each byte to an equivalent character.

Reviewer: Binary strings are supported in V8, we are just adding support for constructing it here.  We also shouldn't get into the implementation details of what V8 function it uses here, just briefly describe what is added and enough to make it clear what it is useful for.

```suggestion
- NewStringFromBytes to support the creation of [binary strings](https://developer.mozilla.org/en-US/docs/Web/API/DOMString/Binary) from a Go byte array.
```
